lab5.cpp

```diff
@@ -0,0 +1,66 @@
+﻿#include <iostream>
+// #include <string>
+
+using namespace std;
+
+class Fighter
+{
+public:
+    string name;
+    int health;
+    int damagePerAttack;
+
+    string getName()
+    {
+        return name;
+    }
+    int getHealth()
+    {
+        return health;
+    }
+    int getDamagePerAttack()
+    {
+        return damagePerAttack;
+    }
+
+    Fighter(string name, int health, int damagePerAttack)
+
+    {
+        this->name = name;
+        this->health = health;
+        this->damagePerAttack = damagePerAttack;
+    }
+};
+
+class Fight
+{
+public:
+    void declareWinner(Fighter fighter1, Fighter fighter2)
+    {
+        while (1)
+        {
+            fighter1.health -= fighter2.damagePerAttack;
+            fighter2.health -= fighter1.damagePerAttack;
+            if (fighter1.health <= 0)
+            {
+                cout << "Winner - " << fighter1.name << endl;
+                break;
+            }
+            else if (fighter2.health <= 0)
```

Comment:
if only two condition, then you can not write the "else if" but can write only "else"

Reply:
else does not match the given condition